templates/internet-of-things/index.html

```diff
@@ -220,6 +245,7 @@ <h2>Ready&ndash;to&ndash;go gateway solutions</h2>
     <div class="col-6">
       <h2>Update Control</h2>
       <p>Regular updates for Ubuntu and your apps protect devices from ongoing attacks. Robust and reliable update mechanisms mean we safely address security issues automatically by default. If you want to review changes, Update Control ensures that changes are only applied after they are certified by the device manufacturer.</p>
+      <p><a href="/core/features/ota-updates">Learn more abouut IoT OTA updates&nbsp;&rsaquo;</a></p>
```

Comment:
What's the thinking behind using the caret symbol at the end of links? We should probably stick with the usual Vanilla patterns (for consistency) and only diverge if there's a very good reason to do so.